### ISSUE TITLE
plot(img;xlims=(-3,3),ylims=(-3,3)) not working in plotly()

### ISSUE THREAD

**Issue by dlfivefifty** [JuliaPlots/Plots.jl#850]
```julia
plotly()
img = rand(RGB{Float64}, 256, 256)
plot(img;xlims=(-3,3),ylims=(-3,3))
```
doesn't match the correct behaviour, as seen with `gr()`.

**Comment by mkborregaard** [comment]
Yes, this is a well-known bug https://github.com/JuliaPlots/Plots.jl/issues/316#issuecomment-263813404 but probably nice to have it's own issue.

**Comment by mkborregaard** [comment]
It's probably not very hard to fix.

**Comment by daschw** [comment]
Closed by #1253
